manifests/pipeline.yaml

```diff
@@ -1,53 +1,67 @@
 apiVersion: v1
 kind: Template
 labels:
-  app: fedora-coreos
-  template: fedora-coreos-template
+  app: ${DISTRO}-coreos
+  template: ${DISTRO}-coreos-template
 metadata:
   annotations:
     description: |-
-      Jenkins pipeline for Fedora CoreOS.
+      Jenkins pipeline for CoreOS distributions.
     iconClass: icon-jenkins
     openshift.io/display-name: Fedora CoreOS Pipeline
     openshift.io/documentation-url: https://github.com/coreos/fedora-coreos-pipeline
     openshift.io/support-url: https://github.com/coreos/fedora-coreos-pipeline
-    openshift.io/provider-display-name: Fedora CoreOS
-    tags: fcos,jenkins,fedora
-  name: fedora-coreos
+    openshift.io/provider-display-name: fedora CoreOS
+    tags: ${TAGS}
+  name: ${DISTRO}-coreos
 parameters:
   - description: Git source URI for Jenkinsfile
     name: REPO_URL
     value: https://github.com/coreos/fedora-coreos-pipeline
   - description: Git branch/tag reference for Jenkinsfile
     name: REPO_REF
     value: master
+  - description: Distribution name
+    name: DISTRO
+    value: fedora
+  - description: Tags to label the pipeline with
+    name: TAGS
+    value: fedora,jenkins,fcos
+  - description: Name of the namespace to use
+    name: NAMESPACE
+    value: fedora-coreos
```

Comment:
Here's what I'd suggest here: drop the `NAMESPACE` parameter completely and drop all the `namespace: fedora-coreos` keys throughout this file. By default, all these objects are created in the current project one is in. So we should be able to abstract the actual namespace name out of this template.

Reply:
Agreed, and dropped. The initial thinking was to change as little as possible and that's why it was made a parameter.